src/snippets/glamorous.js

```diff
@@ -127,35 +127,27 @@ const SubmitButton = glamorous.button(
     marginTop: '1rem'
   },
   props => ({
-    backgroundColor: props.disabled ? '#BBB' : undefined
+    backgroundColor: props.disabled ? '#BBB' : null
   })
-);
-
-SubmitButton.defaultProps = {
-  type: 'submit'
-};
+).withProps({type: 'submit'});
 
 export default class Login extends Component {
-  constructor(props) {
```

Comment:
Yay for getting rid of unnecessary constructors! :tada:

Reply:
I would have done this in all the examples, but I don't think [buble][buble] supports it :(

Wonder if it's worth switching to babel :/ 

[buble]: https://buble.surge.sh/#%2F*%0A%20%20Thanks%20for%20trying%20Bublé!%20Quickstart%3A%0A%0A%20%20%24%20npm%20install%20-g%20buble%0A%20%20%24%20echo%20%22const%20answer%20%3D%20()%20%3D%3E%2042%22%20%3E%20input.js%0A%20%20%24%20buble%20input.js%20%3E%20output.js%0A%20%20%24%20cat%20output.js%0A%0A%20%20This%20is%20a%20relatively%20new%20project%20–%20please%20report%0A%20%20any%20bugs%20you%20find%20by%20clicking%20'report%20bugs'%20and%0A%20%20linking%20to%20a%20version%20of%20this%20page%20demonstrating%0A%20%20the%20issue.%20Thanks!%0A%0A%20%20Note%20that%20some%20ES2015%20features%20are%20deliberately%0A%20%20unsupported%20-%20click%20the%20'guide'%20link%20in%20nav.%0A*%2F%0A%0A%2F%2F%20arrow%20functions%0Aconst%20add%20%3D%20(%20a%2C%20b%20)%20%3D%3E%20a%20%2B%20b%3B%0A%0A%2F%2F%20classes%0Aclass%20Circle%20extends%20Shape%20%7B%0A%20%20state%20%3D%20%7B%0A%20%20%09color%3A%20'blue'%0A%20%20%7D%3B%0A%20%20constructor%20(%20radius%20)%20%7B%0A%20%20%20%20super()%3B%0A%20%20%20%20this.radius%20%3D%20this%3B%0A%20%20%7D%0A%0A%20%20area%20()%20%7B%0A%20%20%20%20return%20Math.PI%20*%20Math.pow(%20this.radius%2C%202%20)%3B%0A%20%20%7D%0A%7D%0A%0A%2F%2F%20object%20literals%0Alet%20obj%20%3D%20%7B%0A%20%20shorthandProperty%2C%0A%20%20shorthandMethod%20()%20%7B%0A%20%20%20%20console.log(%20'hello!'%20)%3B%0A%20%20%7D%2C%0A%20%20%5Bcomputed%5D%3A%2042%0A%7D%3B%0A%0A%2F%2F%20template%20strings%0Alet%20message%20%3D%20%60%0A%20%20hello%20%24%7Bname%7D!%0A%20%20it's%20good%20to%20see%20you%60%3B%0A%0A%2F%2F%20destructuring%0Alet%20%7B%20x%2C%20y%20%7D%20%3D%20point%3B%0Alet%20%7B%20top%2C%20left%20%7D%20%3D%20element.getBoundingClientRect()%3B%0Alet%20%5B%20one%2C%20two%2C%20three%20%5D%20%3D%20document.querySelectorAll(%20'p'%20)%3B%0A%0A%2F%2F%20default%20parameters%0Afunction%20foo%20(%20options%20%3D%20%7B%7D%20)%20%7B%0A%20%20if%20(%20options.bar%20)%20alert(%20options.baz%20)%3B%0A%7D%0A%0A%2F%2F%20rest%20parameters%0Afunction%20sprintf%20(%20str%2C%20...values%20)%20%7B%0A%20%20return%20str.replace(%20%2F%25(w%2B)%2Fg%2C%20(%20match%2C%20type%20)%20%3D%3E%20%7B%0A%20%20%20%20return%20format(%20values.shift()%2C%20type%20)%3B%0A%20%20%7D)%3B%0A%7D%0A%0A%2F%2F%20spread%20operator%0Avar%20max%20%3D%20Math.max(%20...values%20)%3B%0A%0A%2F%2F%20block%20scoping%0Afor%20(%20let%20i%20%3D%200%3B%20i%20%3C%2010%3B%20i%20%2B%3D%201%20)%20%7B%0A%20%20const%20square%20%3D%20i%20*%20i%3B%0A%20%20setTimeout(%20()%20%3D%3E%20console.log(%20square%20)%2C%20i%20*%20100%20)%3B%0A%7D%0A%0A%2F%2F%20binary%20and%20octal%20literals%0A0b101010%20%3D%3D%3D%200o52%3B

Reply:
Ah, gotcha. Sorry, I didn't look at whether you're using babel and just assumed you were.

Reply:
Fair assumption! I am actually working on it now though, so stay tuned :)